joebvp/VPmeasure.py

```diff
@@ -272,6 +272,14 @@ def __init__(self,specfilename,parfilename=None,wave1=None,wave2=None,numchunks=
         cfg.normflux=self.normflux
         cfg.filename=self.specfilename
 
+        # define bad pixels
+        cond_badpix = (self.spectrum.wavelength <= self.spectrum.wvmin) | \
+                      (self.spectrum.wavelength >= self.spectrum.wvmax) | \
+                      (self.spectrum.sig <= 0) | \
+                      (self.spectrum.flux / self.spectrum.sig < cfg.min_sn)  # bad S/N
```

Comment:
I worry about this because we would like to include pixels for lines that are saturated (or damped) and the flux goes black.  Wouldn't this last S/N criterion remove those pixels as well?